src/Microsoft.TemplateSearch.TemplateDiscovery/Filters/SkipTemplatePacksFilter.cs

```diff
@@ -0,0 +1,51 @@
+using Microsoft.TemplateSearch.TemplateDiscovery.PackChecking.Reporting;
+using Microsoft.TemplateSearch.TemplateDiscovery.PackProviders;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.TemplateSearch.TemplateDiscovery.Filters
+{
+    internal class SkipTemplatePacksFilter
+    {
+        private static readonly List<string> _packagesToBeSkipped = new List<string>
+        {
+            "microsoft.dotnet.common.itemtemplates",
+            "microsoft.dotnet.common.projecttemplates",
+            "microsoft.dotnet.test.projecttemplates",
+            "microsoft.dotnet.web.itemtemplates",
+            "microsoft.dotnet.web.projecttemplates",
+            "microsoft.dotnet.web.spa.projecttemplates",
+            "microsoft.dotnet.winforms.projecttemplates",
+            "microsoft.dotnet.wpf.projecttemplates",
+            //"nunit3.dotnetnew.template",
```

Comment:
consider removing commented out code

Reply:
this is an exception - I've added the reason why it is not included.